pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -718,6 +721,72 @@ const (
 	minSafeProxySize = 2
 )
 
+func (cr *PerconaXtraDBCluster) setProbesDefaults() {
+	if cr.Spec.PXC != nil {
+		if cr.Spec.PXC.LivenessInitialDelaySeconds != nil {
+			cr.Spec.PXC.LivenessProbes.InitialDelaySeconds = *cr.Spec.PXC.LivenessInitialDelaySeconds
+		} else if cr.Spec.PXC.LivenessProbes.InitialDelaySeconds == 0 {
+			cr.Spec.PXC.LivenessProbes.InitialDelaySeconds = 300
+		}
+
+		if cr.Spec.PXC.LivenessProbes.TimeoutSeconds == 0 {
+			cr.Spec.PXC.LivenessProbes.TimeoutSeconds = 5
+		}
+
+		cr.Spec.PXC.LivenessProbes.SuccessThreshold = 1
```

Comment:
should we overwrite it? As i understand user can specify his own value

Reply:
yes, please see comment https://github.com/percona/percona-xtradb-cluster-operator/pull/946#discussion_r676372373